src/server/vnparse.c

```diff
@@ -3926,18 +3923,22 @@ do_schedselect(char *select_val, void *server, void *destin, char **presc_in_err
 	if (rc != 0)
 		return rc;
 	while (chunk) {
+#ifdef NAS /* localmod 082 */
+		int k;
+#endif
 		if (firstchunk)
 			firstchunk = 0;
 		else
 			strcat(outbuf, "+");
 
 #ifdef NAS /* localmod 082 */
-		j = (pserver ? pserver->sv_nseldft : 0)  + (pque ? pque->qu_nseldft : 0);
+		k = (pserver ? pserver->sv_nseldft : 0)  + (pque ? pque->qu_nseldft : 0);
 		if (parse_chunk(chunk, j, &nchk, &nelem, &pkvp, &nchunk_internally_set) == 0)
```

Comment:
You replaced 'j' with 'k' but didn't replace where it is being passed into parse_chunk()

Reply:
Good catch, Bhroam! Fixed.